.github/workflows/ci.yml

```diff
@@ -125,7 +126,7 @@ jobs:
           mdbook-version: '0.4.25'
       - name: Emit book logs to tmp.txt, fail if build logs contain 'ERROR'
         run: |
-          MDBOOK_preprocessor__FORC_documenter__STRICT="true" mdbook build docs/book &> tmp.txt
+          MDBOOK_preprocessor__FORC_documenter__STRICT="true" mdbook build docs/book 2>&1 | tee tmp.txt
```

Comment:
Using `tee` instead of `&>` will help debug those issues that only happens on the CI by printing and capturing the `stderr` even when `mdbook` exits with any status code other than `0`.